Packs/ANYRUN/pack_metadata.json

```diff
@@ -2,10 +2,10 @@
     "name": "ANY.RUN",
     "description": "Empowers SOC teams with a Cloud Sandbox for real-time malware analysis, Threat Intelligence Lookup, and high-quality feeds to enhance detection and threat coverage.",
     "support": "partner",
-    "currentVersion": "2.0.0",
+    "currentVersion": "2.0.1",
```

Comment:
it's change that'd be defined as minor, so please change the version to 2.1.0 instead, and change the rn file name as well